docs/sources/image-rendering/troubleshooting.md

```diff
@@ -1,124 +1,26 @@
-
-To install the plugin, refer to the [Grafana Image Renderer Installation instructions](https://grafana.com/grafana/plugins/grafana-image-renderer/?tab=installation).
-
-## Run in custom Grafana Docker image
-
-We recommend setting up another Docker container for rendering and using remote rendering. Refer to [Remote rendering service]({{< relref "#remote-rendering-service" >}}) for instructions.
-
-If you still want to install the plugin in the Grafana Docker image, refer to [Build with Grafana Image Renderer plugin pre-installed]({{< relref "../installation/docker/#build-with-grafana-image-renderer-plugin-pre-installed" >}}).
-
-## Remote rendering service
-
-> Requires an internet connection.
-
-The [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can also be run as a remote HTTP rendering service. In this setup, Grafana renders an image by making a HTTP request to the remote rendering service, which in turn renders the image and returns it back in the HTTP response to Grafana.
-
-You can run the remote HTTP rendering service using Docker or as a standalone Node.js application.
-
-### Run in Docker
-
-The following example shows how to run Grafana and the remote HTTP rendering service in two separate Docker containers using Docker Compose.
-
-Create a `docker-compose.yml` with the following content:
-
-```yaml
-version: '2'
-
-services:
-  grafana:
-    image: grafana/grafana:main
-    ports:
-      - '3000:3000'
-    environment:
-      GF_RENDERING_SERVER_URL: http://renderer:8081/render
-      GF_RENDERING_CALLBACK_URL: http://grafana:3000/
-      GF_LOG_FILTERS: rendering:debug
-  renderer:
-    image: grafana/grafana-image-renderer:latest
-    ports:
-      - 8081
-```
-
-And then run:
-
-```bash
-docker-compose up
-```
-
-## Run as standalone Node.js application
-
-The following example describes how to build and run the remote HTTP rendering service as a standalone Node.js application and configure Grafana appropriately.
-
-1. Clone the [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) Git repository.
-1. Install dependencies and build:
-
-   ```bash
-   yarn install --pure-lockfile
-   yarn run build
-   ```
-
-1. Run the server:
-
-   ```bash
-   node build/app.js server --port=8081
-   ```
-
-1. Update Grafana configuration:
-
-   ```
-   [rendering]
-   server_url = http://localhost:8081/render
-   callback_url = http://localhost:3000/
-   ```
-
-1. Restart Grafana.
-
-## PhantomJS
-
-> Starting from Grafana v7.0.0, all PhantomJS support has been removed. Please use the Grafana Image Renderer plugin or remote rendering service.
-
-## Troubleshoot image rendering
-
-Enable debug log messages for rendering in the Grafana configuration file and inspect the Grafana server log.
+- Enable debug log messages for rendering in the Grafana configuration file and inspect the Grafana server log.
 
 ```bash
 [log]
 filters = rendering:debug
 ```
```

Comment:
Move down to own section "Enable debug mode".

Reply:
Tried something, let me know what you think about this new section.